### PR TITLE
Semantic Segmentation Porting

### PR DESCRIPTION
**Pull request by dustjr1565**
1. Semantic Segmentation Porting in DLT
- metric: mpa (mean pixel accuracy), iou

2. After the training is completed, "artifacts/model.json" will be copied to "configs/model.json" << Please provide feedback on this.
- run dlt test code

### CODE REVIEW

File: tests/test_hub.py [hunk 1]
```diff
@@ -451,6 +447,31 @@ def test_autocare_dlt_text_recognition(text_recognition_dataset: Dataset, tmpdir
...
+        root_dir=tmpdir,
+    )
+    hub = Hub.load(name=name, root_dir=tmpdir)
+    hub: Hub = Hub.from_model_config(
+        name=name + "_from_model_config",
+        model_config_file=tmpdir / name / Hub.MODEL_CONFIG_FILE,
+        root_dir=tmpdir,
+    )
+
+    _total(hub, dataset, image_size, tmpdir)
```
Comment (ZeroAct) [inline tests/test_hub.py:472]: Should remove `pytest.mark.skip`, plz

**Review by ZeroAct — CHANGES_REQUESTED** [review]

File: waffle_hub/dataset/adapter/autocare_dlt.py [hunk 2]
```diff
@@ -94,6 +94,8 @@ def export_autocare_dlt(self, export_dir: Union[str, Path]) -> str:
         _export_autocare_dlt(self, export_dir, train_ids, val_ids, test_ids, [])
     elif self.task == TaskType.INSTANCE_SEGMENTATION:
         _export_autocare_dlt(self, export_dir, train_ids, val_ids, test_ids, [])
+    elif self.task == TaskType.SEMANTIC_SEGMENTATION:
```
Comment (Jiyong-Jeon) [inline waffle_hub/dataset/adapter/autocare_dlt.py:97]: NIT: If it's a code that checks if it's a valid task, I think we can reduce the number of codes.

File: waffle_hub/hub/adapter/autocare_dlt/autocare_dlt_hub.py [hunk 3]
```diff
@@ -75,6 +75,8 @@ def __init__(
             root_dir=root_dir,
         )
 
+        self.model_json_output_path = self.hub_dir / self.CONFIG_DIR / "model.json"
+
```
Comment (Jiyong-Jeon) [inline waffle_hub/hub/adapter/autocare_dlt/autocare_dlt_hub.py:79]: The logic will be modified by Refectoring.

File: tests/test_utils.py [hunk 4]
```diff
@@ -12,8 +12,8 @@
 from waffle_hub.utils.data import resize_image
 from waffle_hub.utils.evaluate import (
     evaluate_classification,
+    evaluate_instance_segmentation,
     evaluate_object_detection,
-    evaluate_segmentation,
 )
```
Comment (Jiyong-Jeon) [inline tests/test_utils.py:17]: We don't have test codes related to 'evalute_instance_segmentation' and 'evalute_semantic_segmentation', can you add them?

**Review by Jiyong-Jeon — APPROVED** [review]

File: waffle_hub/hub/model/wrapper.py [hunk 5]
```diff
@@ -220,6 +221,48 @@ def __call__(
         return parseds
 
 
+class SemanticSegmentationResultParset(ResultParser):
```
Comment (ZeroAct) [inline waffle_hub/hub/model/wrapper.py:224]: `Parset` -> `Parser` 😂

Comment (ZeroAct) [reply waffle_hub/hub/model/wrapper.py:224]: I'll merge this branch after fixing it.

**Review by ZeroAct — APPROVED** [review]

**Pull request merged**